cmd/trace-agent/remote_config.go

```diff
@@ -112,3 +114,12 @@ func getContainerTags(req *http.Request, cfg *config.AgentConfig, provider api.I
 	}
 	return nil
 }
+
+func normalize(configsRequest *pbgo.ClientGetConfigsRequest) {
+	// err is explicitly ignored as it is not an actual error and the expected normalized service
+	// is returned regardless.
+	service, _ := traceutil.NormalizeService(configsRequest.Client.ClientTracer.Service, configsRequest.Client.ClientTracer.Language)
+	env := traceutil.NormalizeTag(configsRequest.Client.ClientTracer.Env)
+	configsRequest.Client.ClientTracer.Env = env
```

Comment:
Just to verify, there's no way this can cause a nil pointer issue at some point right?
e.g. `configsRequest.Client` couldn't be nil s.t. `configsRequest.Client.ClientTracer.Env` causes a panic?

Reply:
No, see https://github.com/DataDog/datadog-agent/blob/d76a6234a8916a7ed4f10ae31c51f0606af67a4e/cmd/trace-agent/remote_config.go#L74